### DISCUSSION TITLE
Is it possible to have pages vertically in Xournal++ in Windows 10?

### DISCUSSION THREAD

**Discussion by LeanderBrinkmeier** [xournalpp/xournalpp#4844] [Q&A] [answered]
Whenever I add a new page it is added on the right. I would prefer if the new page would be added below. Is that possible to toggle in the configuration files or within Xournal++ itself? 

I haven't found a similar discussion after a quick search, but if that question has been answered before, I am sorry and would not mind being redirected.

Thank you very much for helping out!

**Comment by rolandlo** [comment, marked as answer]
Yes, that' easily possible. Use menu `View > Rows/Cols > Set Columns > 1 Column`.

**Reply by LeanderBrinkmeier** [reply]
Thank you!
